src/plugins/data_source_management/public/components/data_source_table/data_source_table.tsx

```diff
@@ -147,6 +149,11 @@ export const DataSourceTable = ({ history }: RouteComponentProps) => {
           <EuiButtonEmpty size="xs" {...reactRouterNavigate(history, `${index.id}`)}>
             {name}
           </EuiButtonEmpty>
+          {index.id === uiSettings.get('defaultDataSource', null) ? (
```

Comment:
Would be nice to have an unit test to cover this change.

Reply:
added